premerge/premerge_resources/main.tf

```diff
@@ -258,6 +258,31 @@ resource "kubernetes_service_account" "windows_2022_object_cache_ksa" {
   depends_on = [kubernetes_namespace.llvm_premerge_windows_2022_runners]
 }
 
+resource "kubernetes_manifest" "linux_runners_pdb" {
+  manifest   = yamldecode(templatefile("pdb.yaml", { runner_set_name : "llvm-premerge-linux-runners", max_pod_count : 16 }))
+  depends_on = [kubernetes_namespace.llvm_premerge_linux_runners]
+}
+
```

Comment:
If we decide to keep 'pdb' as an acronym for 'pod disruption budget' (and I can see why a short acronym is useful), maybe we could at least add a comment here stating what 'pdb' stands for? OR mentioning that the defintions below are for the Kubernetes Pod Disruption Budget (pdb). Something like that.

Reply:
I changed up the names to be `_disruption_budget` to make things more explicit. Never a very bad idea to expand out a (overloaded in the compiler world) acronym.